fuzz/fuzz_helper/src/lib.rs

```diff
@@ -186,7 +188,7 @@ pub fn process_ctap_specific_type(data: &[u8], input_type: InputType) {
     process_message(&command, &mut ctap_state, &mut ctap_hid);
 }
 
-// Splits and reassembles the given data as HID packets.
+// Splits the given data as HID packets and reassembles it, verifying that the original input message is reconstructed.
```

Comment:
This line is 120 characters long. I'm curious why our tests pass. Maybe a workflow is missing a directory here?

I'll approve to unblock, and leave it to @gendx to merge.

Reply:
Let's resolve it here, and investigate in a separate bug.

Reply:
FYI, this is intended as we don't enable the (unstable) `wrap_comments` option of Rustfmt:
- https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#wrap_comments
- https://github.com/rust-lang/rustfmt/issues/3347